doc/en/components/grids/_shared/row-editing.md

```diff
@@ -264,9 +258,10 @@ If you want the buttons to be part of the keyboard navigation, then each on of t
 
  ```ts
 public rowEditActionsTemplate = (ctx: IgcGridRowEditActionsTemplateContext) => {
+    const endRowEdit = ctx.$implicit;
     return html`
-        <button onClick="${this.endRowEdit(false)}">Cancel</button>
-	    <button onClick="${this.endRowEdit(true)}">Apply</button>
+        <button onClick="${endRowEdit(false)}">Cancel</button>
```

Comment:
This throws and error when set to the grid:
![image](https://github.com/IgniteUI/igniteui-xplat-docs/assets/10397980/fea9bb7a-6539-4eab-8d03-2d7b4f3355b2)

Seems to be a problem with the event binding, can be modified to: 
```
    public rowEditActionsTemplate = (ctx: IgcGridRowEditActionsTemplateContext) => {
        const endRowEdit = ctx.$implicit;
        return html`
            <button @click=${(e: any) => endRowEdit(false)}>Cancel</button>
            <button @click=${(e: any) => endRowEdit(true)}>Apply</button>
        `;
    };
```